package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dashevo/insight-api",
   "description": "A Dash blockchain REST and WebSocket API Service",
-  "version": "2.2.9",
+  "version": "2.3.0",
```

Comment:
Don't we need to increase major version if we remove method or it didn't work anyway so we kinda keep backward compatibility? :)))

Reply:
yeah, kind of. Honestly don't know what is best here :). I'm fine with any bump. Which do you prefer? Major, minor or patch?

Reply:
Let’s keep it as is.